Constants-BCC.lua

```diff
@@ -0,0 +1,145 @@
+local _, addon = ...
```

Comment:
DRY, override config keys depending on wow version.

Reply:
Do you mean deleting the -BCC and -Classic file and do "magic" in the one singular Constant file? How would you suggest solving the overwriting of keys most efficiently?

Add a new one for those that must be changed, like both having a addon.CATEGORY_SORT_ORDER and addon.CATEGORY_SORT_ORDER_CLASSIC and switch between them in code? Or do you have some magic I don't know about? :D

Reply:
Why should sort order differ in versions?